.github/actions/ansible_validate_changelog/validate_changelog.py

```diff
@@ -125,6 +125,8 @@ def is_valid_changelog_format(path: str) -> bool:
             changelog_config = yaml.safe_load(config_file)
             changes_type = tuple(item[0] for item in changelog_config["sections"])
             changes_type += (changelog_config["trivial_section_name"],)
+            changes_type += (changelog_config["prelude_section_name"],)
+            changes_type += (changelog_config["new_plugins_after_name"],)
```

Comment:
What does this do?

Reply:
Well, I mean I kinda get what it's doing, I just don't understand the key name here. Might be beyond the scope of this PR though.

Reply:
That is correct, I am not sure what section should be `prelude_section_name`
I"ll take it out!

Reply:
If you take prelude section name out, it wouldn't recognize `release_summary`.

Reply:
@NilashishC My bad, removed `new_plugins_after_name`  now, confused it with `prelude_section_name` while removing did not check config.yml 😿